contents/blog/sql-for-analytics.md

```diff
@@ -0,0 +1,134 @@
+---
+title: The basics of SQL for analytics
+date: 2023-08-31
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/happy-hog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Explainers
+---
+
+SQL is a programming language to help you interact with data. One of SQL’s most important use cases is analytics. In analytics, there are many types of data and ways to slice that data. SQL provides the depth and customizability to get insights from it, answer questions, and make data-driven decisions.
+
+This post goes over what SQL is, how to analyze your analytics data with SQL, and recommendations for your analysis process.
+
+## What is SQL?
```

Comment:
Same comment re above and SQL being common knowledge for engineers. I dont think we should include this section

Reply:
removed